bundles/org.openhab.binding.neeo/README.md

```diff
@@ -44,43 +44,43 @@ Sending OFF to any Recipe status channel does nothing.
 A NEEO Device is simply a collection of Macros that the Device supports.
 
 A NEEO Macro is an action that can be performed on the Device.
-Actions can be triggered by sending ON to the channel
+Actions can be triggered by sending ON to the channel.
 
 ## Supported Things
 
 - Bridge: NEEO Brain.
 This bridge represents a physical NEEO Brain and will contain one to many Rooms within it.
 
 - Bridge: NEEO Room.
-Represents a Room on the NEEO Brain.  Only rooms that have atleast one device or one recipe (custom if no devices) will be shown unless the brain configuration option "discoverEmptyRooms" is set to true.
+Represents a Room on the NEEO Brain. Only rooms that have at least one device or one recipe (custom if no devices) will be shown unless the Brain configuration option "discoverEmptyRooms" is set to true.
 
 - Thing: NEEO Device.
 
 Represents a Device within the NEEO Room.
 
-## Discover
+## Discovery
 
 NEEO Brains will be automatically discovered if mDNS/bonjour/zeroconf is installed on the local machine:
 
 1. On Windows - installing iTunes will install bonjour.
 1. On Linux - please install zeroconf (see vendor documentation on how to do that).
 1. On Mac - should already be installed.
 
-When you add the NEEO Brain, the Rooms on the Brain will then be auto discovered and placed in the inbox.
-When you add a Room, all Devices should be auto discovered and placed in the inbox.
-If you remove any discovered thing either from the inbox or from the added things, simply re-trigger a NEEO binding scan to rediscover it.
+When you add the NEEO Brain, the Rooms on the Brain will be auto-discovered and placed in the Inbox.
+When you add a Room, all Devices should be auto-discovered and placed in the Inbox.
+If you remove any discovered Thing either from the Inbox or from the added Things, simply re-trigger a NEEO binding scan to rediscover it.
 
-If you have the Brain both wired and wireless, the Brain will NOT be discovered twice (only once) and which interface is discovered depends on the timing of the beacon discover message (first one wins).
-If you discovered the wired first but want to use the wireless (or in the reverse), add the Brain and then modify its configuration to the IP address you want to use.
+If the Brain is both wired and wireless, it will NOT be discovered twice (only once). Which interface is discovered depends on the timing of the beacon discovery message (first one wins).
```

Comment:
```suggestion
If the Brain is both wired and wireless, it will NOT be discovered twice (only once).
Which interface is discovered depends on the timing of the beacon discovery message (first one wins).
```